option java_package = "com.mi.game.network.data.bin";

message BINRegisterRequest {
	required string userName = 1;
	required string password = 2;
	optional string fullName = 3;
}

message BINRegisterResponse {
    required bool responseCode = 1;
	optional string message = 2;
}

